docs/LED-INDICATION.md

```diff
@@ -22,3 +22,16 @@ that deserve user attention.
 | 12  | Controller replied without TLS connection state. |
 | 13  | Controller replied without OCSP response. |
 | 14  | Failed to fetch or verify Controller certificate. |
+
+# Appstatus display using LEDs
+
+Starting from commit e31ec53c0dbec104e4818c998223449576585d6b, application status is displayed using LEDs.
+That commit only supports model SIEMENS AG.SIMATIC IPC127E
```

Comment:
Instead of "that commit" use EVE as the third person.

Reply:
"Currently" matches the above approach.